aries-site/src/pages/foundation/color.mdx

```diff
@@ -1,13 +1,147 @@
+
+Applying semantic colors as intended ensures your design:
+
+- Is aligned with the current theme and visual direction.
+- Is accessible.
+- Enables theme mode switching (light and dark).
+- Improves maintenance (when themes are updated, designs are synchronised with token updates. See [design tokens overview](/design-tokens/overview).)
+
+The HPE Design System color palette and guidance differ from Brand Central. Prioritize the HPE Design System palette when selecting colors for app or web experiences.
+
+## Color palettes
+
+Color expresses brand, guides interaction, and supports accessibility. It shapes interface elements, typographic emphasis, data visualization clarity, and adaptation between light and dark modes. Applied intentionally, it reinforces inclusive design across HPE applications and web experiences.
+
+
+### Backgrounds
+Background colors establish depth, hierarchy, and cognitive grouping. Products typically begin with `color.background.back` as the base, then layer surfaces using `color.background.front` and `color.background.floating` to convey elevation. Use contrast background tokens to separate dense regions, preferring contrast backgrounds over borders in tight layouts to reduce visual noise.
+
+For more direction on how to use background colors, see [background colors guidance](/foundation/background-colors-guidance).
+
+<BackgroundSwatch />
+
+#### Overlay
+
+Overlay color dims inactive content to direct attention to an active layer; apply it behind modal, drawer, or dialog surfaces to reduce cognitive load using `color.background.overlay`.
+
+### Borders
+Border colors delineate areas only when spacing or background contrast are insufficient. Use `color.border.default` for neutral boundaries and dividers, and omit borders where existing spacing or contrast already provides clear separation.
+
+<BorderSwatch />
+
+### Text
+Text colors help create hierarchy and ensure readability. Use `color.text.default` for body copy and `color.text.heading` for headings. `color.text.strong` can be used to emphasize key figures or other important text. To de-emphasize content and create a clear visual hierarchy, apply `color.text.weak` to secondary information.
+
+### Icon
+Icon colors follow text hierarchy and must remain legible at small sizes. Match an icon's color emphasis with its accompanying text.
+
+### Decorative
+Decorative colors provide brand and accent without conveying status or interaction. Limit secondary decorative hues to illustration and non‑interactive ornamentation, and avoid substituting decorative colors for status; use semantic text or foreground status tokens instead.
+
+<DecorativeSwatch />
+
+### Data visualization
+Dataviz colors differentiate categories without overwhelming perception.
```

Comment:
Should we communicate a bit more about how dataviz colors are designed to  ensure categorical data series are visually distinct and recognizable from one another?

```suggestion
Dataviz colors differentiate categories without overwhelming perception. 

This palette has been crafted to ensure each series is visually distinct and color blind friendly. Apply dataviz colors according to the following guidelines:
```